### DISCUSSION TITLE
cpp cublas

### DISCUSSION THREAD

**Discussion by Policing2024** [openai/whisper#2159] [General]
I have

https://github.com/ggerganov/whisper.cpp/releases/download/v1.5.4/whisper-cublas-12.2.0-bin-x64.zip

https://huggingface.co/ggerganov/whisper.cpp/resolve/main/ggml-large-v3-q5_0.bin!

how to use it with subtitle edit pls

**Comment by phineas-pta** [comment]
export transcription as `.srt` file
